### PR TITLE
VZ-8678: Update the instructions in Argo CD sample

### PR DESCRIPTION
**Pull request by bhatpmk**

### CODE REVIEW

**Review by MelwynPaul — APPROVED** [review]
Looks good. 
Thank you.

File: content/en/docs/samples/argo-cd.md [hunk 1]
```diff
@@ -62,17 +62,17 @@ This example provides information about how to deploy the `Hello-helidon` applic
...
     - **Repository URL**: https://github.com/verrazzano/verrazzano/
     - **Revision**: `{{<release_branch>}}`
     - **Path**: Path in the repository where the Kubernetes resource definitions are listed. For example: `examples/helidon-config`
 7. Under the **Destination** section, do the following:
     - **Cluster URL**: Select the cluster to which you want to deploy the applications.
-    - **Namespace**: Specify the namespace in which you want to deploy the applications.
+    - **Namespace**: Specify the namespace in which you want to deploy the applications. The instructions in this sample use `hello-helidon` as the namespace.
 9. Click **Create**.
-10. If you selected `Manual` as the **Sync Policy** option, then click **Sync**.
+10. If you selected `Manual` as the **Sync Policy** option, then click **Sync**. In the **Synchronize Resources** section, select all the resources and then click **Synchronize**.
```
Comment (rosemarymarano) [inline content/en/docs/samples/argo-cd.md:75]: -> resources, (comma) and

**Review by rosemarymarano — APPROVED** [review]
LGTM, one very minor edit

File: content/en/docs/samples/argo-cd.md (hunk 1 above)
Comment (bhatpmk) [reply content/en/docs/samples/argo-cd.md]: Fixed

**Review by desagar — APPROVED** [review]

**Pull request merged**